### ISSUE TITLE
[Feature Request] Make Podcast from local files

### ISSUE THREAD

**Issue by ohare93** [akhilrex/podgrab#236]
Title says it all. I have the local files that make up a podcast like show, and wish to stream it as such to my devices. 

I use this app to download real podcasts out there on the internet, and then sync them to my phone via the rss link. It works great.

I simply wish I could do the same for other items :+1:

**Comment by kifty** [comment]
This would allow me to integrate my podcast listening and my audiobook listening into my podcast player, which would be super convenient!

**Comment by ohare93** [comment]
@kifty I switched over to https://github.com/advplyr/audiobookshelf as it does both. Though the podcast tracking and scheduled downloading are not as great.

**Comment by tehniemer** [comment]
@ohare93 why now use both? This for downloading and the other for serving, that's going to be my plan.

**Comment by ohare93** [comment]
@tehniemer a few points on Audiobookshelf:
1. It *can* download podcasts. It's just a little weird about it as you need to match the show to one online then setup a schedule for each show to check. I much prefer Podgrab in this regard, but it works fine so far.
2. It can serve local podcast files without a link to particular podcast. This is great for shows that no longer exist, or just files of people talking that were never actually podcasts.
3. User authentication is better. This is necessary if you want to serve your shows as an rss feed for your other devices out to the web.

**Comment by tehniemer** [comment]
Right, my point was that podgrab is the better downloader, so keep using it to download and use audiobookshelf for everything else.

**Comment by n8willis** [comment]
Not to pile on here on become a plus1-er, but I have run into a use case for this feature that I haven't seen come up yet: DAB recordings from a local-radio-tuner (or even SDR app).

Often the metadata is available from schedule data or the RDS channel and the apps can create robust MP3s that are perfectly usable in a standard audio player app, but they are hard to integrate with other sources.

Possibly the big example here would be something like TVheadend, which can schedule recordings with ease, metadata included, but the user has to then manually move the files around. This is recently a big new point of argument in the UK, as the BBC has started trying to push people to install its siloed "BBC Sounds" app to get its podcasts; the standard RSS feed versions of the same shows are delayed by a month.

(So it's technically possible to subscribe to those feeds if you want to hear what was new a month in the past, or you can grab them out of the air when they're current, but then you have the sync problem. Having Podgrab watch a directory or the like and serve the local files would be a significant boon.)
